modules/ROOT/pages/to-request-access.adoc

```diff
@@ -4,13 +4,25 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-APIs that have been protected by a Client ID enforcement policy require API consumers to create an application to hold the Client ID and Secret. This lets the app access the API as well as provide consumer analytics for an API developer.
+When you request access, the client application is registered and creates an
+API contract between the API and the client.
 
-If terms and conditions are set for an asset, you are prompted to accept the terms before access can be granted.
+APIs that are protected by a client ID enforcement policy require
+applications to have a client ID and client secret. This enables the
```

Comment:
APIs that are protected by a Client ID Enforcement Policy require client application to provide the client ID and client secret credentials.

*the client ID and client secret credentials are automatically created when the client application is registered.*

Reply:
Added this info.